tests/test_path.py

```diff
@@ -37,3 +41,45 @@ def test_path(system_Haldane_PythTB):
                                                   [3, 3, 3],
                                                   [4, 4, 4],
                                                   [5, 5, 5],]) / 10), "path.K_list is wrong"
+
+
+    quantities = quantities_tab
+
+    tab_result = wberri.tabulate(system = system_Haldane_PythTB,
+                    grid = path,
+                    quantities = quantities,
+                    user_quantities = {},
+                    parallel=None,
+                    parameters = {"external_terms":False},
+#                    specific_parameters = specific_parameters,
+                ibands = [0],
+                use_irred_kpt = True , symmetrize = True, # should have no effect, but will check the cases and give a warning
+#                parameters_K = parameters_K,
+#                frmsf_name = None,
+#                degen_thresh = degen_thresh, degen_Kramers = degen_Kramers
+                )
+
+    filename = "path_tab.pickle"
+    fout = open(os.path.join(OUTPUT_DIR, filename),"wb")
+    pickle.dump(tab_result,fout)
+    tab_result_ref = pickle.load(open(os.path.join(REF_DIR, filename),"rb") )
```

Comment:
Instead of saving the whole results object as pickled file (which can break when we change the structure of result class), I would prefer to save only the data.

For example:
```python3
data = {}
for quant in ["E"]+quantities:
    for comp in get_component_list(quant):
        data[f"{quant}.{comp}"] = tab_result.results.get(quant).get_component(comp)

with open(os.path.join(REF_DIR, filename), "rb") as f:
    pickle.dump(data, f)
```

Reply:
Done